docs/installation-deployments.md

```diff
@@ -8,6 +8,41 @@ npx @portkey-ai/gateway
 
 # Deploy to Cloudflare Workers
 
+Method 1
+
+1. Create an API token in Cloudflare
+Select "My Profile" from the dropdown menu of your user icon on the top right of your dashboard. Select "API Tokens" > "Create Token". Under "Custom Token", select "Get started". Name your API token in the "Token name" field. Under "Permissions", select "Account", "Cloudflare Pages" and "Edit". Then "Continue". Now your Cloudflare API token is created. 
+
+2. Clone the repo
```

Comment:
This step looks unnecessary in context of docs. We can replace this with something like this:
```
Add github workflow .yml file to your forked version of the repo under .github/workflows directory. Here is a sample cloudflare-deploy.yml file that will deploy to cloudflare on every push to main branch.
```

What do you think?